### ISSUE TITLE
Error with nonexistent variable when running study on v3.3.0

### ISSUE THREAD

**Issue by a-paxton** [Dallinger/Dallinger#844]
When running [one of my Dallinger experiments](https://github.com/thomasmorgan/joint-estimation-game) in Dallinger v3.3.0, I get a long traceback (see below) referencing a nonexistent vector property, which appears to have been solved [earlier this year](https://github.com/Dallinger/Dallinger/pull/777).

## Expected Behavior
The game runs perfectly on v3.4.1 on commit 5240190.

## Current Behavior
The game won't run:
```16:46:41 web.1      |  2018-01-14 16:46:41,760 Validating WebSocket request
16:46:41 web.1      |  [2018-01-14 16:46:41,770] ERROR in app: Exception on /node/3/vectors [GET]
16:46:41 web.1      |  Traceback (most recent call last):
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/flask/app.py", line 1982, in wsgi_app
16:46:41 web.1      |      response = self.full_dispatch_request()
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/flask/app.py", line 1614, in full_dispatch_request
16:46:41 web.1      |      rv = self.handle_user_exception(e)
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/flask/app.py", line 1517, in handle_user_exception
16:46:41 web.1      |      reraise(exc_type, exc_value, tb)
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
16:46:41 web.1      |      rv = self.dispatch_request()
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
16:46:41 web.1      |      return self.view_functions[rule.endpoint](**req.view_args)
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/dallinger/experiment_server/experiment_server.py", line 817, in node_vectors
16:46:41 web.1      |      return success_response(vectors=[v.__json__() for v in vectors])
16:46:41 web.1      |    File "/anaconda3/envs/dallinger__3_4_1/lib/python2.7/site-packages/dallinger/models.py", line 1362, in __json__
16:46:41 web.1      |      "info_id": self.info_id,
16:46:41 web.1      |  AttributeError: 'Vector' object has no attribute 'info_id'
16:46:41 web.1      |  2018-01-14 16:46:41,774 Closing Dallinger DB session at flask request end
```

## Steps to Reproduce (for bugs)
1. Roll back Dallinger to v3.3.0.
2. Clone + start [the experiment](https://github.com/thomasmorgan/joint-estimation-game) in sandbox mode.
3. Play through training for both players.
4. After training, both players should see an "error" screen.
5. Check the Heroku Papertrail for the above error.
Note: You'll get similar results if you're running it locally in debug, but the errors will pop up in your local Terminal.

## Your Environment
* Dallinger version used: 3.3.0
* Operating System and version (desktop or mobile): iOS High Sierra 10.13.12
* Browser and version (e.g. Chrome 39, node.js 5.4): Chrome 63.0.3239.132
* Link to your project: https://github.com/thomasmorgan/joint-estimation-game

**Comment by suchow** [comment]
This is fixed in 3.4.1.

@a-paxton What is the error that is preventing you from upgrading to 3.4.1? (i.e., when you try running this on 3.4.1, what breaks?)

**Comment by a-paxton** [comment]
Right now, when I run on `master`, I get this error from my Chrome console:

```
Uncaught ReferenceError: Fingerprint2 is not defined
    at create_participant (dallinger.js:175)
    at instruct-1?hit_id=ZZ2HRR&assignment_id=27YJLJ&worker_id=5VWMHU&mode=debug:15
create_participant @ dallinger.js:175
(anonymous) @ instruct-1?hit_id=ZZ2HRR&assignment_id=27YJLJ&worker_id=5VWMHU&mode=debug:15
instruct-1?hit_id=ZZ2HRR&assignment_id=27YJLJ&worker_id=5VWMHU&mode=debug:39 Uncaught ReferenceError: proceedWithParticipantID is not defined
    at HTMLButtonElement.onclick (instruct-1?hit_id=ZZ2HRR&assignment_id=27YJLJ&worker_id=5VWMHU&mode=debug:39)
onclick @ instruct-1?hit_id=ZZ2HRR&assignment_id=27YJLJ&worker_id=5VWMHU&mode=debug:39
```

But I opened this because I was under the impression that you wanted me to report this, based on our conversation last night. Feel free to delete it if I misunderstood!
